containers/Ideas/Idea.js

```diff
@@ -1,5 +1,13 @@
 import React, { useState, useEffect, useRef } from 'react'
-import { Alert, View, Image, ScrollView, Platform, KeyboardAvoidingView, Pressable } from 'react-native'
+import {
+  Alert,
```

Comment:
While you are at that: can we also order the imports alphabetically?
And maybe for the imports also have a rule like 
1. React
2. react-native
3. other stuff
4. own stuff - components
5. own stuff - styles
(Just as an idea for a rule, but I find the imports a bit hard to read if we have many.)